php-checks/src/main/java/org/sonar/php/checks/formatting/IndentationCheck.java

```diff
@@ -54,15 +54,15 @@ private void checkImplementListIndentation(FormattingStandardCheck formattingChe
       int classDecLine = node.getTokenLine();
       int expectedColumn = getLineStartingColumn(node) + PSR2_INDENTATION;
 
-      if (!isOnSameLine(node.getToken(), Iterables.getLast(interfaceList).getToken())) {
+//      if (!isOnSameLine(node.getToken(), Iterables.getLast(interfaceList).getToken())) {
 
         if (!isCorrectlySplittedOnLines(classDecLine, interfaceList)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AIndentationCheck)